### ISSUE TITLE
Crud should not close when user clicks outside Modal

### ISSUE THREAD

**Issue by mvorisek** [atk4/ui#1378]
make it even default? (`X` close icon must continue to work on user click)

**Comment by bedengler** [comment]
I would make that selectable. 
Sometimes it's good, sometimes not. 
If you show just some info it's good. 
If you open a form where the user enters something it could be bad usability if it's closed by accident and the user has to re-enter everything again.

If it isn't closed automatically there has to be the closing function using a "X" added by default for convenience...

**Comment by mkrecek234** [comment]
I agree with bedengler - most users I have appreciate the background tap = close functionality: They use it on tablets and that's a learnt and appreciated behaviour. Otherwise you have to point to the X, and that is less convenient that any point on the (larger) background space. Unintended close of forms where you entered data should be protected by ->canLeave = false, so you have to confirm closing not to lose data
